public/components/common/form/hooks.tsx

```diff
@@ -0,0 +1,86 @@
+import { useState } from 'react';
+import _ from 'lodash';
```

Comment:
According to our [STYLEGUIDE](https://github.com/wazuh/wazuh-kibana-app/blob/master/STYLEGUIDE.md#modules) it says that we should write `import { isEqual } from 'lodash';` as far as I understand it